cfme/tests/services/test_catalog.py

```diff
@@ -31,6 +34,9 @@ def test_catalog_duplicate_name():
     cat.delete()
 
 
+# Un-collecting for 5.8 because of BZ: 1486041
+@pytest.mark.uncollectif(lambda: current_version() < '5.9')
```

Comment:
REQUIRED: please use BZ instead

NB: please don't mix several fixes in one PR

Reply:
@izapolsk  actually this is a fix for 5.9, but in 5.8 this test case fails due to above BZ.
The problem with the above BZ is that it's marked as verified, so here BZ can't be used.

Reply:
@anurag03, 
1. there should be clone of this BZ for 5.8 in Bugzilla then. please ask Simaishi about clone.
2. replace current_version with  lambda: store.current_appliance.version